settings.gradle

```diff
@@ -0,0 +1,5 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+rootProject.name = 'cyclonedx-gradle-plugin'
```

Comment:
By default, gradle will use the directory name as the project name.  Sometimes this causes weird things to happen if people map the directory name to something custom.  This sets the project name explicitly to avoid such problems.